vulture_os/toolkit/api_parser/proofpoint_trap/proofpoint_trap.py

```diff
@@ -194,6 +203,9 @@ def execute(self):
         # fetch at most 24h of logs to avoid the process running for too long
         to = min(timezone.now(), since + timedelta(hours=24))
 
+        # delay the times of 15 minutes, to get the "updated" event
+        to = to - timedelta(minutes=15)
```

Comment:
this will cause problems when last_api_call is still 1 minute before and 'to' is then set **before** 'since'